artifactory/services/utils/multipartupload.go

```diff
@@ -49,8 +49,6 @@ const (
 
 	// API constants
 	uploadsApi        = "/api/v1/uploads/"
```

Comment:
Please run `go fmt ./...`
```suggestion
	uploadsApi = "/api/v1/uploads/"
```